src/use/phone/useScripts.js

```diff
@@ -28,27 +36,28 @@ export default ({ callType }: UseScriptsProps) => {
 
   const scriptHeaders = {
     // note: translated later, no $t here.
-    [CallType.INBOUND]: ['~~This person is calling you ', '~~(inbound-call)'],
+    [CallType.INBOUND]: [
+      'phoneDashboard.inbound_description',
+      'phoneDashboard.inbound_call_parenthesis',
+    ],
     [CallType.OUTBOUND]: [
-      '~~You are calling this person back.',
-      '~~(call-back)',
+      'phoneDashboard.callback_description',
+      'phoneDashboard.callback_parenthesis',
     ],
     [CallType.CALLDOWN]: [
-      '~~This person has case you calling to check if they still need help.',
-      '~~(call-down)',
+      'phoneDashboard.calldown_description',
+      'phoneDashboard.calldown_parenthesis',
       [
         {
-          title: '~~YES',
-          body:
-            '~~You are still on the map. There are no guarantees. It may take a long time.',
-          note: '~~(Add a note about your call)',
+          title: 'phoneDashboard.if_yes',
```

Comment:
Went ahead and fixed this locale key. Assuming this is what you meant?

Reply:
useScripts.js looks good.
Good catch on all_remaining_callbacks. I'm pretty sure that was an autocomplete issue in Spreadsheets.